fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -111,6 +111,14 @@ std::string getErrorReason(BackgroundErrorReason reason) {
 	}
 }
 
+ACTOR Future<Void> forwardError(Future<int> input) {
```

Comment:
Any reason to forward the error here?

Reply:
If we send an `Error` directly from within `RocksDBErrorListener`,  the constructor will crash.